OKiMaureen/OnlineTuts
# ft(landing-page): create landing page

#### What does this PR do?
Install Angular and Angular frontend app
install Ngrx


#### Description of Task to be completed?
 Create the landing page and auth page

#### How should this be manually tested?
- Fetch and check out to this branch with `git fetch  ft-landing-page-168766191&& git checkout ft-landing-page-168766191
-  Open Browser and make a  request to `http://localhost:4200` to view the created landing page
-  Open Browser and make a  request to `http://localhost:4200/signup` to view the created sign up page
 Open Browser and make a  request to `http://localhost:4200/signin` to view the created sign-in page

#### On success 
```
You would see the corresponding pages.

```
#### What are the relevant pivotal tracker stories?
[##168766191](https://www.pivotaltracker.com/story/show/#168766191)
